test/webtest/src/test/java/com/onfido/qa/websdk/test/WorkflowIT.java

```diff
@@ -0,0 +1,74 @@
+package com.onfido.qa.websdk.test;
+
+import com.onfido.qa.websdk.mock.TaskDefinition;
+import com.onfido.qa.websdk.mock.WorkflowRun;
+import com.onfido.qa.websdk.page.BasePage;
+import com.onfido.qa.websdk.page.CountrySelector;
```

Comment:
we have combine the CountrySelector and DocumentTypeSelector, this test should actually fail. use RestrictedDocumentSelection instead.

Reply:
Won't fail, if the element that it expects to find is present. Which is the case. I'll anyway update it to RestrictedDocumentSelection